build.sbt

```diff
@@ -276,3 +276,13 @@ lazy val freestyleHttpFinch = (project in file("http/finch"))
       %%("finch-core")
     )
   )
+
+lazy val freestyleHttpAkka = (project in file("http/akka"))
+  .dependsOn(freestyleJVM)
+  .settings(name := "freestyle-http-akka")
+  .settings(
+    libraryDependencies ++= Seq(
+      "com.typesafe.akka" %% "akka-http" % "10.0.5",
```

Comment:
Some help needed: ¿how can we do to use the `%%%` notation for  `akka-http`? Note that, no matter the name, its version number is different from that of `akka`.